### PR TITLE
fix(client): update cert logo and other layout changes

### LINKED ISSUE #37892: Update logo on certifications
Our certifications are still using the old version of the freeCodeCamp logo (with the underscore under the fire). 

They should instead use our new logo, which can be found at https://github.com/freecodecamp/design-style-guide

### PR DESCRIPTION
**Pull request by ahmaxed**
This PR:

- Updates the certification view logo to the current design.
- Updates the padding on the intro and donation section button groups.

Closes #37892

### CODE REVIEW

File: client/src/client-only-routes/ShowCertification.js [hunk 1]
```diff
@@ -198,20 +199,13 @@ class ShowCertification extends Component {
 
     return (
       <div className='certificate-outer-wrapper'>
-        {conditionalDonationSection}
+        <Grid>{conditionalDonationSection}</Grid>
```
Comment (ojeytonwilliams) [inline client/src/client-only-routes/ShowCertification.js]: Same point as `Intro`, this already has a `Grid` element.  How about replacing the inner `Grid` with a `div`?

I guess this is *probably* fine as is, I'm just not sure if there's a side-effect of nesting containers that I just haven't noticed.

**Review by ojeytonwilliams — APPROVED** [review]
Other than hand-wringing about nesting Grids, everything LGTM.  I'm okay with it as is, but I thought I'd mention it.

**Comment by raisedadead** [comment]
Hi @ahmadabdolsaheb can you take a look.

**Comment by ahmadabdolsaheb** [comment]
@ojeytonwilliams thank you for the review. I added the grid for the side padding. 
I addressed your points, removed grid for one of the instances, and added padding for the other so it should be good to go.

**Pull request merged**